trade-finance-manager-ui/templates/utilisation-reports/_macros/report-reconciliation-status.njk

```diff
@@ -6,8 +6,8 @@
 
   {% set tagColourClassForStatus = {
     'REPORT_NOT_RECEIVED': 'govuk-tag--red',
-    'PENDING_RECONCILIATION': '',
-    'RECONCILIATION_IN_PROGRESS': 'govuk-tag--blue',
+    'PENDING_RECONCILIATION': 'govuk-tag--blue',
+    'RECONCILIATION_IN_PROGRESS': 'govuk-tag--light-blue',
```

Comment:
There is an equivalent component in portal that also needs updating please: https://github.com/UK-Export-Finance/dtfs2/blob/main/portal/templates/utilisation-report-service/previous-reports/_macros/report-reconciliation-status.njk

Reply:
Thanks @RLavender98 great shout - updated.

FYI, pending feedback from design RE status tags. In the meantime, I've just added "light-blue" so there are 2x different "blue tags".